src/stores/SetupEncryptionStore.ts

```diff
@@ -255,6 +261,9 @@ export class SetupEncryptionStore extends EventEmitter {
                     },
                     setupNewCrossSigning: true,
                 });
+
+                await initializeDehydration(true);
+
```

Comment:
@richvdh what do you mean by unrecoverable here? the session or something else?

Reply:
errrr I don't entirely remember. I think that I meant we won't have a dehydrated device set up for this account (and maybe no way to create one?).

@uhoreg do you remember if an issue was created for this?

Reply:
No dehydrated device will be created.  So if there was no previous dehydrated device, there will not be one.  If there was a previous dehydrated device, the old one will remain, which means it will be encrypted with the old SSSS key.  So if the user doesn't have the old SSSS key (which is probably why they reset SSSS), they won't be able to rehydrate.

It doesn't have an issue specifically for it, but it's part of https://github.com/element-hq/element-web/issues/27336